src/lib.rs

```diff
@@ -60,20 +60,14 @@
 //!   - Includes signals
 //!   - Integrates with `std::process` and `std::io::Error`
 //!   - Doesn't integrate with `main`
-//! - [exit-code][exit-code]
-//!   - `i32` constants and helper methods
-//!   - Doesn't include signals
-//!   - Doesn't integrate with `main`, `std::process`, or `std::io::Error`
-//! - [exitcode][exitcode]
-//!   - `i32` constants and helper methods
-//!   - Doesn't include signals
-//!   - Doesn't integrate with `main`, `std::process`, or `std::io::Error`
+//! - [sysexits][sysexits]
+//!   - Only includes exit codes
+//!   - Uses an exit code tagged enum which can be converted to [`std::process:ExitCode`]
 //! - [exitfailure][exitfailure]
 //!   - Allows `Display`able errors to be used with [`?` in `main()`](https://github.com/rust-lang/rust/issues/43301)
```

Comment:
I think I'd rather have a note about their status.   For example, `exitcode` still has a lot of dependents and has a high download count.  It is reasonable to still compare with it.

Reply:
exit-code definitely not since it has 2K all time downloads, but yeah lets include exitcode.

Reply:
Please restore the `exitcode` section that was in here

Reply:
Even if it's "deprecated"? I purposefully excluded the link so that people are disentivised from using it.

Reply:
Yes, as explained above.  "its still reasonable to compare it"

Reply:
Actually, I just looked at each recommendation and I don't understand them. `sysexit` and `exit-code` have both been downloaded only 2K times without an update in 4 years. `sysexit` has literally no crates using it while `exit-code` has one dependent (which hasn't been updated in 2 years). `exitcode` is fine except it's also abandoned and doesn't support Termination. Similarly, `exitfailure` is ok, but is specifically designed for `failure` which has been superseded by a number of other error handling libraries. In short, all of these recommendations seem either bogus or out-of-date. I'd argue the alternatives should just be sysexits with perhaps anyhow (and/or error_stack).

I personally see "related crates" as recommendations, so it doesn't make sense to offer out-of-date or unused crates IMO. Feel free to close if you disagree.

Reply:
>  f they were recommended, then this crate wouldn't exist.

I guess by reading "related" as "recommended," I meant "if this doesn't fit your use case, here's this other thing that might." But anyway, saying `Crates considered when making this one include` makes it super clear what you meant now, thanks.